### DISCUSSION TITLE
Something went wrong when I try to edit a file online

### DISCUSSION THREAD

**Discussion by victor-ponz-ies-el-caminas** [community/community#84055] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi.
When I try to edit any file in my repos (even in repos that I collaborate) it shows a message
`Error
Looks like something went wrong!`
My user is [https://github.com/victor-ponz-ies-el-caminas/](https://github.com/victor-ponz-ies-el-caminas/)
Thank you

**Comment by twr14152** [comment]
I have the same issue. 
User https://github.com/twr14152/

Update I can update README file in repo from mobile app but still unable to from Web UI.
